lib/src/combined_wrappers.dart

```diff
@@ -0,0 +1,73 @@
+// Copyright (c) 2017, the Dart project authors.  Please see the AUTHORS file
```

Comment:
I'd prefer this file just be `combined_list.dart`, possibly in a `combined_wrappers` directory.